internal/wal_uploader.go

```diff
@@ -168,12 +208,13 @@ func uploadBulkMetadata(uploader *WalUploader, walFilePath string) {
 		}
 	}
 	dtoBody, _ := json.Marshal(walMetadataArray)
```

Comment:
Why ignore the error here and above?
```
file, _ := ioutil.ReadFile(walMetadataFile)
walMetadataFiles, _ := filepath.Glob(walMetadataBulkFolder.GetFilePath("")...

```

Reply:
There was still open discussion about  whether we need to fail if the wal-metadata upload fails for any reason (missing file/upload failure etc). I will change all the function calls based on the decision.